pocs/utils/logger.py

```diff
@@ -45,25 +46,34 @@ def get_root_logger(profile='panoptes', log_config=None):
     # Get log info from config
     log_config = log_config if log_config else load_config('log').get('logger', {})
 
+    invoked_script = os.path.basename(sys.argv[0])
     log_dir = '{}/logs'.format(os.getenv('PANDIR', '/var/panoptes/'))
```

Comment:
Should we assume /var/panoptes exists? Or use the python determined temporary directory, i.e. tempfile.gettempdir()?

Reply:
We assume `$PANDIR` exists. It is checked [here](https://github.com/panoptes/POCS/blob/develop/pocs/__init__.py#L71) before any logging can be loaded. So really we don't even need the `getenv` call but it was already there. Might not be a bad idea to switch it to a temp file but shouldn't really every get there.